packages/trace-viewer/src/ui/actionList.tsx

```diff
@@ -129,7 +151,7 @@ export const renderAction = (
   return <>
     <div className='action-title' title={action.apiName}>
       <span>{action.apiName}</span>
-      {locator && <div className='action-selector' title={locator}>{locator}</div>}
+      <ActionParameterContext action={action} sdkLanguage={sdkLanguage || 'javascript'} />
```

Comment:
Style nit: we don't get paid per line, so given no immediate reuse I would inline.

```suggestion
      {parameterString && <div
        className={clsx('action-parameter', parameterString.type === 'locator' && 'action-locator-parameter')}>
          {parameterString.value}
       </div>}
```